### PR TITLE
Adds the TP-23

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Adds the TP-23 heavy pistol, another secondary option for the marines. And replaces the 1911 in req with this. Has less capacity and fire rate, higher damage.

## Why It's Good For The Game
I feel like another option isn't bad. This also replaces a random "service" pistol with another one that makes more sense.

## Changelog
:cl:
add: Adds the TP-23 heavy pistol.
balance: Replaces the m1911 with the TP-23
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -14. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/projectiles/guns/pistols.dm [hunk 1]
```diff
@@ -99,6 +99,37 @@
 	accuracy_mult = 1.15
 	damage_mult = 1.1
 
+//-------------------------------------------------------
+//M1911
```
Comment (psykzz) [inline code/modules/projectiles/guns/pistols.dm:103]:
```suggestion
// TP-23 service pistol
```

**Comment by psykzz** [comment]
Actually can you add both the old and new pistol to the appropriate tier spawner.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-15). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Comment by saprasam** [comment]
this is an official solid snake gun

**Pull request merged**